rules/_deprecated/persistence_creation_modif_launch_deamon_sequence.toml

```diff
@@ -1,8 +1,8 @@
 [metadata]
 creation_date = "2020/12/07"
-integration = ["endpoint"]
-maturity = "production"
-updated_date = "2025/02/04"
+deprecation_date = "2025/03/19"
```

Comment:
We need to change the name of the rule in the current releae, push and then wait for release cycle before we deprecate. Sample https://github.com/elastic/detection-rules/pull/4537